web/impact/impact/settings.py

```diff
@@ -197,8 +197,8 @@ class Base(Configuration):
     V0_SITE_NAME = bytes(os.environ.get(
         'IMPACT_API_V0_SITE_NAME', 'masschallenge.org'), 'utf-8')
 
-    V0_API_GROUP = bytes(os.environ.get(
-        'IMPACT_API_V0_API_GROUP', 'v0_clients'), 'utf-8')
+    V0_API_GROUP = os.environ.get(
```

Comment:
The changes look good. Did we get to know what is causing this(`bytes`) to fail now and why is it still working for V0_SITE_NAME and V1_API_GROUP?

Reply:
I am not certain why in the first place it was changed to bytes but under v1 if the user is an employee, then the permission is granted which is not given for v0. Sorry, pushing a commit to update v1 as well.